src/content/mods.cpp

```diff
@@ -84,7 +89,9 @@ void parseModContents(ModSpec &spec)
 			mod_conf_has_depends = true;
 			std::string dep = info.get("optional_depends");
 			dep.erase(std::remove_if(dep.begin(), dep.end(),
-					static_cast<int(*)(int)>(&std::isspace)), dep.end());
+						  static_cast<int (*)(int)>(
+								  &std::isspace)),
```

Comment:
Ditto.

Reply:
This was done by clang-format, please fix that instead